google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITTransactionManagerAsyncTest.java

```diff
@@ -224,6 +222,8 @@ public void testRollback() throws InterruptedException {
     }
   }
 
+  @Ignore(
```

Comment:
Changes in this class don't immediately appear specific to this PR. Can you break this into a separate PR?

Reply:
It seems that that had already been done. I've brought this and the `commit-stats2` up to date with the latest changes from `master`, and there is are differences anymore in this class between this branch and `master`.